epub34/authoring/index.html

```diff
@@ -5864,40 +5833,51 @@ <h4>The <code>itemref</code> element</h4>
-						refer only to XHTML content documents and their authoring requirements, a number of which need
-						resolving (e.g., providing an <a href="#sec-epub-type-attribute">alternative to the
-								<code>epub:type</code> attribute</a>). These references and requirements will be updated
-						once it is clearer that support will definitely be added in this revision. Until then, the
-						specification should be read as also supporting HTML wherever it currently refers to XHTML,
-						despite the support inconsistencies.</p>
-
-					<p>To provide feedback on this change, please add comments to <a
-							href="https://github.com/w3c/epub-specs/issues/2715">issue 2715</a> in the GitHub
-						tracker.</p>
-				</div>
+				<h3>XHTML content documents</h3>
 
 				<section id="sec-xhtml-intro" class="informative">
 					<h4>Introduction</h4>
 
-					<p>This section defines a profile of [[html]] for creating HTML content documents. An instance of an
-						HTML document that conforms to this profile is a [=core media type resource=].</p>
+					<p>An [=XHTML content document=] is an instance of an <a href="#sec-xml-constraints">XML
+							document</a> expressed using the <a data-cite="html#the-xml-syntax">XML syntax</a> of the
+						[[[html]]] [[html]].</p>
+
+					<p>XHTML content documents that conform to this profile are [=core media type resources=] and one of
+						the two formats that constitute [=EPUB content documents=] &#8212; that can be used without
+						fallbacks in the [^spine^].</p>
+
+					<div class="note">
+						<p>EPUB 3 does not support the <a data-cite="html#the-html-syntax">HTML syntax</a> of [[html]]
+							as a core media type. Although both serializations allow the same elements to be expressed,
+							not all JavaScript libraries and APIs are designed to handle the XML syntax, in particular
+							the shadow DOM. This means, for example, that although the [^template^] element [[html]] can
+							be authored in the XML syntax it may not be usable in the same way as in the HTML
+							syntax.</p>
+
+						<p>The Publishing Maintenance Working Group is aware that the HTML Standard no longer recommends
+							the use of the XML syntax. XML, however, is integral to many publishing workflows and
+							expected in many vendor ingestion systems and reading systems. It is also not expected that
+							support for XHTML will be dropped from browsers, even if there are additional support
+							differences between the syntax in the future. Support for the HTML syntax will be addressed
+							in a successor format to EPUB 3.</p>
+					</div>
+
+					<div class="note">
+
+						<p>Earlier versions of EPUB 3 referred to W3C's HTML5 standard to define EPUB content documents,
+							but that standard was largely a snapshot of the HTML Standard maintained by the WHATWG.
+							After reaching an agreement on the development of HTML in 2018, the W3C standard was retired
```

Comment:
```suggestion
							After reaching an <a href="https://www.w3.org/2019/04/WHATWG-W3C-MOU.html">agreement</a> on the development of HTML in 2019, the W3C standard was retired
```

I presume that is the agreement you referred to...